### PR TITLE
Speed up members

### PR DESCRIPTION
**Pull request by Br3nda**

### CODE REVIEW

File: app/controllers/harvests_controller.rb [hunk 1]
```diff
@@ -21,7 +21,7 @@ def index
     end
 
     @harvests = Harvest.search('*', where:    where,
-                                    limit:    100,
+                                    limit:    Harvest.per_page
```
Comment (milesgould) [inline app/controllers/harvests_controller.rb]:
```suggestion
                                    limit:    Harvest.per_page,
```

File: app/controllers/members_controller.rb [hunk 2]
```diff
@@ -7,9 +7,20 @@ class MembersController < ApplicationController
 
   def index
     @sort = params[:sort]
-    @members = members
+
+    order = if @sort == 'recently_joined'
```
Comment (milesgould) [inline app/controllers/members_controller.rb]: You could factor this out as a method, which should make CodeFactor happy again.

File: app/models/concerns/search_members.rb [hunk 3]
```diff
@@ -0,0 +1,56 @@
+# frozen_string_literal: true
+
+module SearchMembers
+  extend ActiveSupport::Concern
+
+  included do
+    searchkick merge_mappings: true,
```
Comment (milesgould) [inline app/models/concerns/search_members.rb]: This file looks fine to me, I'd be in favour of just disabling the CodeFactor warnings on this.

**Comment by Br3nda** [comment]
gone down a rabbithole/yakshave getting permissions working on the jsonapi. Currently, on this branch, anyone can edit anyone's data -- so don't merge this. Need to get it checking with cancancan.

**Comment by maco** [comment]
@Br3nda I just made this PR officially a draft, given your warning.

**Pull request closed without merging**